clean_project/search_engine.py

```diff
@@ -1,16 +1,10 @@
-"""
-Search a word in a file and return all the occurrences specified the line numbers
-"""
 import argparse
-import psycopg2
-
 from index import load_config_file
-
-BOLD = '\033[1m'
-END = '\033[0m'
+import psycopg2
+import search_cli
```

Comment:
there shouldn't be a dependency to search_cli. the other modules should depend on search_engine, not the other way around. please get rid of this